docs/admin/cluster-management.md

```diff
@@ -169,8 +169,8 @@ This is an infrequent event, but it requires careful management. There is a sequ
    1. Update existing objects in the storage to new version by running `cluster/update-storage-objects.sh`.
    1. Turn off the old API version.
 
+<a name="Turn-on-or-off-an-api-version-for-your-cluster"></a>
```

Comment:
Looks like the syntax is just to strip the punctuation from the header and then delimit all the words with hyphen (-) characters.

In this case you should just be able to update the link with "#Turn-on-or-off-an-api-version-for-your-cluster" without having to add the explicit <a name> tag at the target.